.github/workflows/CI2.yml

```diff
@@ -108,14 +115,32 @@ jobs:
             -DCMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }}
           cmake --build build --parallel 2
           cmake --install build
+      - name: Checkout arborx
+        if: ${{ matrix.arborx }}
+        uses: actions/checkout@v2.2.0
+        with:
+          repository: arborx/ArborX
+          ref: master
+          path: arborx
+      - name: Build arborx
+        if: ${{ matrix.arborx }}
+        working-directory: arborx
+        run: |
+          cmake -B build \
+            -DCMAKE_PREFIX_PATH=${HOME}/kokkos
```

Comment:
```suggestion
            -DKokkos_ROOT=${HOME}/kokkos
```

Reply:
Does it matter? Both should do the same.

Reply:
It does not matter.  Just my preference ;)